Dockerfile

```diff
@@ -74,6 +77,8 @@ COPY --from=builder /opt/netbox/venv /opt/netbox/venv
 
 ARG NETBOX_PATH
 COPY ${NETBOX_PATH} /opt/netbox
+# Copy the modified 'requirements*.txt' files, to have the files actually used to install
```

Comment:
`[…] used to install` what? I think you're either missing some words to finish the sentence or the formulation is wrong-ish.


```suggestion
# Copy the modified 'requirements*.txt' files, to have the files actually used during installation
```